### PR TITLE
Foo

### PR DESCRIPTION
**Pull request by kugel-**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/kugel-/ReviewNinja-Welcome/pull/2" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -39,3 +39,5 @@ That’s it! These are the basic features of ReviewNinja to help you get
 started. We hope you enjoy using the tool, and please 
 [let us know](https://github.com/reviewninja/review.ninja/issues/new) 
 how we can make it better.
+
+FFS
```
Comment (kugel-) [inline ReadMe.md]: !fix Could be improved

Comment (kugel-) [inline ReadMe.md]: okay

Comment (kugel-) [inline ReadMe.md]: Not bad! but try harder

Comment (kugel-) [inline ReadMe.md]: meh

Comment (kugel-) [inline ReadMe.md]: !fixed

**Pull request merged**